modules/apps/foundation/frontend-theme/space-theme/src/templates/navigation.ftl

```diff
@@ -0,0 +1,63 @@
+
+<nav class="${nav_css_class}" id="navigation" role="navigation"  class="navbar navbar-default navbar-fixed-top">
+	<div class="container-fluid">
+		<h1 class="hide-accessible"><@liferay.language key="navigation" /></h1>
```

Comment:
You have the contents of the ```<h1 />``` on a single line here, but right below it you have it on separate lines.  We should be consistent and put everything on separate lines.